repository/src/main/resources/alfresco/subsystems/Authentication/identity-service/identity-service-authentication.properties

```diff
@@ -12,6 +12,7 @@ identity-service.resource=alfresco
 identity-service.credentials.secret=
 identity-service.public-client=true
 identity-service.admin-console.redirect-path=/alfresco/s/admin/admin-communitysummary
+identity-service.webscript-home.redirect-path=/alfresco/s/index

```

Comment:
Kindly check to add this in the release notes if we get the approval to implement this.